python/flask/secure-cookies/session-secure-cookies-dict-set.py

```diff
@@ -0,0 +1,11 @@
+import flask
+
+# ok
+app = flask.Flask()
+
+app.config["SESSION_COOKIE_SECURE"]=True
+app.config["SESSION_COOKIE_HTTPONLY"]=True
+app.config["SESSION_COOKIE_SAMESITE"]='Lax'
```

Comment:
Can you add more tests with different ordering of the config setting?

Reply:
Good idea!